webservices/args.py

```diff
@@ -380,13 +380,13 @@ def make_seek_args(field=fields.Int, description=None):
 
 schedule_b = {
     'committee_id': fields.List(IStr, description=docs.COMMITTEE_ID),
-    'recipient_committee_id': fields.List(IStr, description='The FEC identifier should be represented here the contributor is registered with the FEC'),
+    'recipient_committee_id': fields.List(IStr, description='The FEC identifier should be represented here if the contributor is registered with the FEC'),
     'recipient_name': fields.Str(description='Name of recipient'),
     'disbursement_description': fields.Str(description='Description of disbursement'),
     'recipient_city': fields.List(IStr, description='City of recipient'),
     'recipient_state': fields.List(IStr, description='State of recipient'),
-    'last_disbursement_date': fields.Date(missing=None, description='Filter for records before this date'),
-    'last_disbursement_amount': fields.Float(missing=None, description='Filter for records'),
+    'last_disbursement_date': fields.Date(missing=None, description='Filters for records before this date'),
```

Comment:
Looking at these, it looks like the oned that start with "last" had the wrong explanation, this is used for pagination when you sort by something